Testscripts/Linux/dpdkPmdsBasicCheck.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+# This script will run DPDk basic testpmd command in different PMDs and check failures.
+# To run this script constants.sh details must.
+#
+########################################################################################################
+
+HOMEDIR=$(pwd)
+LOGDIR="${HOMEDIR}/DpdkTestPmdLogs"
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+DPDK_UTIL_FILE="./dpdkUtils.sh"
+FAILSAFE_FILE="./failsafe.sh"
+
+. ${CONSTANTS_FILE} || {
```

Comment:
Not sure if this is required or mandatory.

Reply:
Thanks! I changed it to [ -e $CONSTANTS_FILE ] to confirm the constants.sh exists. The UtilsInit will source it.